src/overlaybd/registryfs/registryfs.cpp

```diff
@@ -334,7 +347,10 @@ class RegistryFSImpl : public RegistryFS {
         DEFER({ release_cURL(req); });
         photon::net::StringWriter writer;
         if (!username.empty()) {
-            req->set_user_passwd(username.c_str(), password.c_str()).set_redirect(3);
+            // req->set_user_passwd(username.c_str(), password.c_str()).set_redirect(3);
```

Comment:
What are the differences from the previous version? does set_user_passwd also use basic auth？

Reply:
set_user_passwd will always set basic auth and libcurl->reset() does not work.